### PR TITLE
Created first draft of bio

### PR DESCRIPTION
**Pull request by veteater**
This is the optional description.

This is for #16.

### CODE REVIEW

File: bios/veteater-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## My Bio
+- Location: San Diego
```
Comment (veteater) [inline bios/veteater-bio.md:2]: Where in SD?

Comment (veteater) [inline bios/veteater-bio.md:2]: will add.

Comment (kathiefoley) [inline bios/veteater-bio.md:2]: What are your hobbies?

Comment (veteater) [inline bios/veteater-bio.md:2]: I love finding new restaurants. I'll add that.

**Comment by veteater** [comment]
:ship: Merging now.

**Pull request merged**